pkg/test/masteruserrecord/assertion.go

```diff
@@ -0,0 +1,98 @@
+package murtest
```

Comment:
why do you name this package `murtest` and not `masteruserrecord_test` as we would normally do?

Reply:
because it's too long. in addition, it's not a test but assertion - you shouldn't use `_test` suffix in that case

Reply:
@MatousJobanek I missed the fact that it's a utility pkg for the tests, and we can't use the `_test` suffix for that. But I'd like to have the pkg name that matches the directory name, really. I think it's a good convention. We can always use an alias when importing the pkg somewhere else.